### PR TITLE
Fix CalibFormats/SiStripObjects comp failure for gcc10

### PR DESCRIPTION
**Pull request by mrodozov**
#### PR description:

CalibFormats/SiStripObjects failed to compile in 2300 gcc10 IB 
it was missing the header for uint_32

#### PR validation:

CalibFormats/SiStripObjects compiles

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
please test for CMSSW_12_0_X_2021-06-09-2300/slc7_amd64_gcc10

**Comment by smuzaffar** [comment]
gcc10 Ibs are broken, so better to test itf or master IB and integrate if test passed

**Comment by mrodozov** [comment]
I think the gcc900 will be enough for this change

File: CalibFormats/SiStripObjects/interface/SiStripDetInfo.h [hunk 1]
```diff
@@ -23,7 +23,7 @@
 // user include files
 #include <map>
 #include <vector>
-
+#include <stdint.h>
```
Comment (makortel) [inline CalibFormats/SiStripObjects/interface/SiStripDetInfo.h:26]: Shouldn't this be
```suggestion
#include <cstdint>
```
(as in #34077)?

**Comment by mrodozov** [comment]
please test

**Comment by smuzaffar** [comment]
@cms-sw/alca-l2 , can you please sign this?
@silviodonato , @qliphy this is just a technical fix to get gcc10 builds error free

**Comment by francescobrivio** [comment]
+alca
 - technical

**Comment by qliphy** [comment]
+1

**Pull request merged**